assets/src/room.ts

```diff
@@ -206,41 +207,8 @@ export class Room {
   }
 
   public init = async () => {
-    await this.askForPermissions();
-
-    // Refresh mediaDevices list after ensuring permissions are granted
-    // Before that, enumerateDevices() call would not return deviceIds
-    const mediaDevices = await navigator.mediaDevices.enumerateDevices();
-    const videoDevices = mediaDevices.filter(
-      (device) => device.kind === "videoinput"
-    );
-
-    for (const device of videoDevices) {
-      const constraints = {
-        video: {
-          ...VIDEO_TRACK_CONSTRAINTS,
-          deviceId: { exact: device.deviceId },
-        },
-      };
-
-      try {
-        this.localVideoStream = await navigator.mediaDevices.getUserMedia(
-          constraints
-        );
-
-        break;
-      } catch (error) {
-        console.error("Error while getting local video stream", error);
-      }
-    }
-
-    try {
-      this.localAudioStream = await navigator.mediaDevices.getUserMedia({
-        audio: AUDIO_TRACK_CONSTRAINTS,
-      });
-    } catch (error) {
-      console.error("Error while getting local audio stream", error);
-    }
+    this.localVideoStream = await getVideoStream();
+    this.localAudioStream = await getAudioStream();
```

Comment:
This breaks the ask-only-once behaviour, so pls create a Jira story to bring it back

Reply:
Done, MV-150